clients/drcachesim/simulator/caching_device_stats.cpp

```diff
@@ -69,6 +73,7 @@ caching_device_stats_t::caching_device_stats_t(const std::string &miss_file,
     stats_map_.emplace(metric_name_t::MISSES, num_misses_);
     stats_map_.emplace(metric_name_t::HITS_AT_RESET, num_hits_at_reset_);
     stats_map_.emplace(metric_name_t::MISSES_AT_RESET, num_misses_at_reset_);
+    stats_map_.emplace(metric_name_t::COMPULSORY_MISSES, num_compulsory_misses_);
```

Comment:
How about also printing this out as a sub-category of misses in the cache simulator?

Reply:
Please add a test: augment the unit tests, and/or add an end-to-end test w/ golden output.

Reply:
I have added suitable print and unit test.